dsl/scripts/jobs.groovy

```diff
@@ -143,6 +143,90 @@ def jobs = [
             stringParam('DEVOPSSCRIPTS_BRANCH', 'main', 'Git branch, tag or hash of the vegaprotocol/devopsscripts repository')
             stringParam('ANSIBLE_BRANCH', 'master', 'Git branch, tag or hash of the vegaprotocol/ansible repository')
         }
+    ],
+    [
+        name: 'private/Deployments/Veganet/Stagnet',
+        useScmDefinition: false,
+        definition: {
+            cps {
+                script("""
+                @Library('vega-shared-library@main') _
+                pipelineDeploy()
+                """)
+            }
+        },
+        env: [
+            NET_NAME: 'stagnet',
+        ],
+        parameters: {
+            stringParam('VEGA_CORE_VERSION', 'develop', "Git branch, tag or hash of the vegaprotocol/vega repository. Leave empty to not deploy a new version of vega core. If you decide not to build binary by yourself you need to set version according to the versions available on releases page: https://github.com/vegaprotocol/vega/releases")
+            booleanParam('DEPLOY_CONFIG', true, 'Deploy some Vega Network config, e.g. genesis file')
+            booleanParam('BUILD_VEGA_CORE', true, 'Decide if VEGA_CORE_VERSION is to be build or downloaded')
+            choiceParam('RESTART', ['YES', 'YES_FROM_CHECKPOINT', 'NO'], 'Restart the Network')
```

Comment:
I think we need to set `YES_FROM_CHECKPOINT` as a default one for all non-Devnet networks. (i.e. put it first in the list).